doc-content/enterprise-only/springboot/bus-app-run-managed_con.adoc

```diff
@@ -0,0 +1,60 @@
+[id='bus-apps-run-managed_{context}']
+
+= Running business application in development mode
+
+Development (managed) mode enables developers to work on a Red Hat Business Automation business application business assets project and dynamically deploy changes to the business application without the need to restart it. In addition, development mode provides a complete monitoring environment for business automation capabilities, for example process instances, tasks, and jobs.
+
+.Prerequisite
+* You have a `<business-application>.zip` file that contains a business assets project, that you created using the http://start.jbpm.org[business applications] web site.
+* You configured the business application.
+* {PRODUCT} {CENTRAL} is installed and running.
+
+.Procedure
+. Navigate to the `<business-application>/<business-application>-service` folder.
+. Enter one of the following commands:
++
+[cols="50%,50%"]
+|===
+|Command
+|Description
+
+|`./launch-dev.sh clean install`
+|Launches in development mode on Linux or Unix.
+
+|`./launch-dev.bat clean install`
+|Launches in development mode on Windows.
+
+|`./launch-dev.sh clean install -Pmysql`
+|Launches in development mode on Linux or Unix if you have configured for a MySQL database.
+
+|`./launch-dev.bat clean install -Pmysql`
+|Launches in development mode on Windows if you have configured for a MySQL database.
+
+|`./launch-dev.sh clean install -Ppostgres`
+|Launches in development mode on Linux or Unix if you have configured for a postgreSQL database.
+
+|`./launch-dev.bat clean install -Ppostgres`
+|Launches in development mode on Windows if you have configured for a PostgreSQL database.
+
+|===
++
+The `clean install` argument directs Maven to build a fresh installation. The projects are then built in the following order:
++
+* Data model
+* Business assets
+* Service
++
+The first time that you run the script, it might take a while to build the project because all dependencies of the project are downloaded. At the end of the build, the application starts.
+. Enter the following command to access your business application:
++
+[source]
+----
+http://localhost:8090/
+----
+. Enter the credentials `user`/`user`  or `kie-server`/`kieserver!`.
+Comment: what's the diff between the two if they both have the kieserver role?
```

Comment:
user is more an application user while kie-server is sort of an admin user. Anyway, these are just sample users that should not be used in real applications